### ISSUE TITLE
Plugin not working with Chaos Mesh v2.0.1.

### ISSUE THREAD

**Issue by leoncard** [chaos-mesh/chaos-mesh-datasource#36]
**What version of Chaos Mesh Datasource are you using?**
0.2.2

**What version of Chaos Mesh are you using?**
2.0.1

**What did you do?**

* I had Chaos Mesh 1.1.2 installed and the plugin working.

* I've upgraded Chaos Mesh to version 2.0.1 and the plugin stopped working, now giving `Annotation Query Failed` errors.

* Just to be sure, I tried pointing the datasource to a dashboard with Chaos Mesh version 1.1.2 and it worked. 

So, there might be an incompatibility with Chaos Mesh version 2.

**What did you expect to see?**
Grafana chaos mesh datasource plugin working with no errors

**What did you see instead?**

Grafana is giving Annotation Query Failed errors.

AnnotationSrv.query error TypeError: l.sent(...).data.map is not a function
    at t.eval (DataSource.ts:157)
    at eval (tslib.es6.js:103)
    at Object.eval [as next] (tslib.es6.js:132)
    at i (tslib.es6.js:74)
(anonymous) @ annotations_srv.ts:89

**Comment by leoncard** [comment]
Hello, this has been here for a while :)  .

Does anyone have an idea on why this is happening? 

Is there any update you are going to do soon on the plugin to be compatible with version 2?

**Comment by g1eny0ung** [comment]
Sorry guys for the late reply.

After #37 merged, you can use it with Chaos Mesh's latest version. But due to https://github.com/chaos-mesh/chaos-mesh/pull/2218 won't be merged into the `release-2.0` branch, So the new version of the plugin will be synchronized with Chaos Mesh `2.1` in the future.

However, we have not yet released it to the official grafana plugin (But we'll get to it soon), so for now you can download it from https://github.com/chaos-mesh/datasource/releases/tag/v2.1.0 if you want to use it.

**Comment by Vikas-Tamboli** [comment]
What version of Chaos Mesh Datasource are you using?
0.2.2

What version of Chaos Mesh are you using?
2.0.4

What did you do?

I've Chaos Mesh version 2.0.4 and the plugin not working, giving Annotation Query Failed errors.

So, there might be an incompatibility with Chaos Mesh version 2.

What did you expect to see?
Grafana chaos mesh Datasource plugin working with no errors for chaos mesh version 2.0.4

What did you see instead?

Grafana is giving Annotation Query Failed errors.

AnnotationSrv.query error TypeError: l.sent(...).data.map is not a function
at t.eval (DataSource.ts:157)
at eval (tslib.es6.js:103)
at Object.eval [as next] (tslib.es6.js:132)
at i (tslib.es6.js:74)
(anonymous) @ annotations_srv.ts:89

Can you please tell me when the compatible plugin will be available for chaos mesh version like 2.0.4

**Comment by g1eny0ung** [comment]
> Can you please tell me when the compatible plugin will be available for chaos mesh version like 2.0.4

@Vikas-Tamboli No version is currently compatible with `2.0.x`. 😢 I'm sorry about this because I don't have enough energy to support these versions.

**Comment by g1eny0ung** [comment]
Since Chaos Mesh has been released `v2.2.0` for a long time and a draft new version Data Source has also been released. I will close this issue.

Thank you all for using and please upgrade to the latest stable releases if possible.  If you still encounter some problems, feel free to open a new issue to describe them.
